wp-content/themes/pub/wporg-learn-2024/inc/head.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Handles adding/removing hooks as needed.
+ */
+function init() {
+	add_filter( 'document_title_parts', __NAMESPACE__ . '\document_title' );
+	add_filter( 'document_title_separator', __NAMESPACE__ . '\document_title_separator' );
+}
+
+/**
+ * Filters document title to add context based on what is being viewed.
+ *
+ * @param array $parts The document title parts.
+ * @return array The document title parts.
+ */
+function document_title( $parts ) {
+	global $wp_query;
+
+	$parts['site']  = __( 'Learn WordPress', 'wporg-learn' );
+	$post_type      = get_query_var( 'post_type' );
+	$sep            = '–';
+
+	if ( is_front_page() ) {
+		$parts['title'] = 'There\'s always more to learn';
```

Comment:
Should this use the site tagline to make it configurable? Or is that not translatable...

![Screenshot 2024-07-15 at 4 27 09 PM](https://github.com/user-attachments/assets/397d2d6d-a1e5-4d3a-b8fa-2eb1ac80df52)

Reply:
Yeah, I'm not sure about the translation, which I think would just follow the set tagline, won't change with translation. I noticed the [developer](https://github.com/WordPress/wporg-developer/blob/trunk/source/wp-content/themes/wporg-developer-2023/inc/head.php#L38) specified it, so I wrote one here. In this case, if the tagline has content, it will extend to `There’s always more to learn | Tagline | Learn.WordPress.org`

Alternatively, we can completely follow the site title and tagline settings and not hardcode it, meaning we can remove the front page condition here. This way, we would get `Site title | Tagline | Learn.WordPress.org`

![image](https://github.com/user-attachments/assets/16a9d9a7-2b3c-4b98-ba84-8dd7e36ee788)
![image](https://github.com/user-attachments/assets/9124ee7c-979e-4290-9e3c-bd7e663caf89)

Reply:
Removed the front page check first here: b348a33